### DISCUSSION TITLE
Ability to Hide Marketplace Sidebar in YAML Web Editor

### DISCUSSION THREAD

**Discussion by dja557** [community/community#44917] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Please provide an option/button to hide the GitHub Actions marketplace sidebar while editing a YAML file on the web. 

Not being able to utilize the full window while editing a YAML file on the web is very inconvenient, and I do not use the marketplace sidebar anyways.

Thank you!

<img width="1483" alt="image" src="https://user-images.githubusercontent.com/87444642/213785094-80f8c597-c736-4ef4-bcd9-cd0fca290509.png">

**Comment by aangelinsf** [comment]
I'm currently looking for a greasemonkey/tampermonkey script that does this and found this request. No luck so far. Might have to roll my own.
